buildpackage.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * This is solely a build script, intended to prep the twin packages for publishing.
+ */
+
+const fs = require("fs");
+const path = require("path");
+const rimraf = require("rimraf");
+const copydir = require("copy-dir");
+const execSync = require("child_process").execSync;
+
+// AzDevOps Package Constants
+const azDevOpsPackageName = "@azurepipelines/azdevops-kube-summary";
+const azDevOpsWebPackConfigFileName = "azdevops.webpack.config.js";
+const azDevOpsPackageFolderName = "azDevOpsPackage";
+
+// WebApp Package Constants
+const webAppPackageName = "@azurepipelines/webapp-kube-summary";
+const webAppWebPackConfigFileName = "webapp.webpack.config.js";
+const webAppPackageFolderName = "webAppPackage";
+
+prepPackage(azDevOpsPackageFolderName, azDevOpsWebPackConfigFileName, editAzDevOpsPackageJson);
+prepPackage(webAppPackageFolderName, webAppWebPackConfigFileName, editWebAppPackageJson);
+
+function prepPackage(packageFolderName, webpackConfigFileName, editPackageJsonCallback) {
+    const binFolderPath = path.join(__dirname, "_bin");
+    ensureDirExists(binFolderPath);
+    
+    const packageFolderPath = path.join(binFolderPath, packageFolderName);
+    console.log(`Deleting folder: ${packageFolderName}`);
+    rimraf.sync(packageFolderPath);
+
+    console.log(`Creating folder: ${packageFolderName}`);
+    fs.mkdirSync(packageFolderPath);
+    fs.statSync

```

Comment:
statsSync, purpose is to check fir exists, it is mentioned acessSync() seems to be the right one
https://nodejs.org/api/fs.html#fs_fs_access_path_mode_callback